apps/codelab/src/app/codelabs/not-found/not-found.component.scss

```diff
@@ -0,0 +1,53 @@
+.container {
```

Comment:
You could technically drop .container wrapper and use :host insted?

Reply:
Yes, it's good idea! Done